skrub/_dataframe/_common.py

```diff
@@ -532,37 +572,6 @@ def _is_numeric_polars(column):
     return column.dtype.is_numeric()
 
 
-@dispatch
-def to_numeric(column, dtype=None, strict=True):
```

Comment:
note the question was about the `numeric_transformer`, not converting strings to numeric dtypes, so `to_numeric` wouldn't be affected, but I think that's what you meant!

Reply:
As discussed IRL, there are pros and cons to keep it:
- pros: we might want to clip specifically numeric values for instance
- cons: a standard scaler (the typical `numeric_transformer`) would be more useful on all numerical outputs rather than numerical inputs